README.md

```diff
@@ -14,6 +14,7 @@ A performant priority queue implementation using a Heap data structure.
 * [import](#import)
 * [API](#api)
   * [constructor](#constructor)
+  * [.from](#fromentries)
```

Comment:
can you add to v4 as well? https://github.com/datastructures-js/priority-queue/tree/v4